sushi-config.yaml

```diff
@@ -29,6 +29,14 @@ dependencies:
 # │  To use a provided input/includes/menu.xml file, delete the "menu" property below.             │
 # ╰────────────────────────────────────────────────────────────────────────────────────────────────╯
 menu:
-  Home: index.html
+  IG Home: index.html
   Table of Contents: toc.html
+  Background:
+    NCPI FHIR: background.html
+    Use Cases: use_cases.html
+    Authors: authors.html
+    Contact: contact.html
   Artifacts: artifacts.html
+  Support: 
+    FHIR Spec: http://hl7.org/fhir/R4/index.html
+    Downloads: downloads.html
```

Comment:
Maybe, this is the place where the mismatching section numbering can be adjusted?